--- sip/cpp/sip_propgridwxPGPropArgCls.cpp.orig	Mon Nov 11 19:40:55 2019
+++ sip/cpp/sip_propgridwxPGPropArgCls.cpp	Sat Mar  5 23:22:46 2022
@@ -317,7 +317,7 @@
             return sipGetState(sipTransferObj);
         }
         else if (sipPy == Py_None) {
-            *sipCppPtr = new wxPGPropArgCls(reinterpret_cast< wxPGProperty * >(NULL));
+            *sipCppPtr = new wxPGPropArgCls(static_cast< wxPGProperty * >(NULL));
             return sipGetState(sipTransferObj);
         }
         else {
